Automattic/wp-calypso
# E2E: replace `﻿calypsoPreReleaseUser`.

#### Changes proposed in this Pull Request

This PR replaces the user for `calypsoPreReleaseUser` with a new user.

For context, see pbOQVh-1p2-p2.

#### Testing instructions

1. pull the branch.
2. decrypt the secrets.
3. ensure the user under `calypsoPreReleaseUser` is replaced with a new one with suffix 2.
